pkg/apis/build/v1beta1/build_types.go

```diff
@@ -193,6 +195,34 @@ func (buildSpec *BuildSpec) StrategyName() string {
 	return buildSpec.Strategy.Name
 }
 
+// VulnerabilityIgnoreOptions refers to ignore options for vulnerability scan
+type VulnerabilityIgnoreOptions struct {
+
+	// ID references the security issues to be ignored in vulnerability scan
+	ID []string `json:"id,omitempty"`
+
+	// Severity indicates the severities of security issues to be ignored (comma separated)
+	Severity *string `json:"severity,omitempty"`
```

Comment:
https://github.com/shipwright-io/build/pull/1489#discussion_r1507188355

I think the implementation is correct now just not the documentation here and in the md.